datafusion/physical-expr/src/aggregate/array_agg.rs

```diff
@@ -37,6 +37,7 @@ pub struct ArrayAgg {
     name: String,
     input_data_type: DataType,
     expr: Arc<dyn PhysicalExpr>,
+    is_expr_nullable: bool,
```

Comment:
What do you think about calling this just `nullable` ? it is a bit more concise and aligns with the name used in `Field`